dts/arm/st/u5/stm32u535Xb.dtsi

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2025 Harris Tomy
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <mem.h>
+#include <st/u5/stm32u535.dtsi>
+
+/ {
+	sram0: memory@20000000 {
+		/* SRAM1 + SRAM2 */
+		reg = <0x20000000 DT_SIZE_K(256)>;
+	};
+
+	sram1: memory@28000000 {
+		/* SRAM4, low-power background autonomous mode */
+		reg = <0x28000000 DT_SIZE_K(16)>;
+	};
```

Comment:
Sorry for this late comment.
These 2 nodes (the 3 SRAMs) are common to all stm32u53/54 SoCs.
Maybe they could be factorized in **stm32u535.dtsi**.